### PR TITLE
Include build date in `application.js` header.

### PR DESCRIPTION
**Pull request by leftclickben**
As suggested.

### CODE REVIEW

**Comment by leftclickben** [comment]
Now also includes environment.

**Comment by leftclickben** [comment]
````
/**
 * rwahs-research-frontend
 * SPA frontend for CollectiveAccess SimpleAPI
 * 
 * @version 0.0.1
 * @date 2016-06-20T09:43:20.935Z
 * @environment staging
 * @link https://github.com/rwahs/research-frontend#readme
 * @license MIT
 */
````

File: gulpfile.js [hunk 1]
```diff
@@ -17,6 +17,8 @@
             ' * ${pkg.description}\n' +
             ' * \n' +
             ' * @version ${pkg.version}\n' +
+            ' * @date ' + (new Date()).toISOString() + '\n' +
```
Comment (kehh) [inline gulpfile.js]: `@date` and `@environment` don't appear in the [jsdoc list of tags ](http://usejsdoc.org/index.html). Does that matter?

Comment (leftclickben) [inline gulpfile.js]: Nah, it's not jsdoc, which describes the functionality (parameters, return type, etc). It just has a syntactic resemblance to jsdoc. `@link` is already there and isn't one either. I did notice a `@copyright` tag which I might use to identify RWAHS but I can make that a separate PR if we want it, this just gives us an easy way to verify deployments.

**Pull request merged**